styles/variables-gallium.less

```diff
@@ -163,9 +163,9 @@
 @agate-panel-h-padding: extract(@agate-panels-panel-padding, 2);
 @agate-panels-partial-width: 890px;
 @agate-panels-breadcrumb-width: 105px;
-@agate-panels-controls-top: (extract(@agate-panels-panel-padding, 1) + 41px);
+@agate-panels-controls-top: (extract(@agate-panels-panel-padding, 1) - 3px);
```

Comment:
We might have to define `@agate-panels-panel-padding` first, then `@agate-panels-controls-top` so that `@agate-panels-panel-padding` variable in this line could be referred properly based on skin?

Reply:
There are same issues above in other skins.

Reply:
I changed thois in `variables-gallium.less` and `variables-silicon.less`. Let me know if I missed something.